NAS documentation (#50)
joyprojects/wildfire: README.md

  - [Instructions](#Instructions)
  - [Requirements](#Requirements)
- [goes/](#goes/)

> - [NAS](#NAS)

Author: Maybe adding the subsections too? like `Storage`

Reviewer: Added!

## goes/

Utilties for using NOAA's GOES-16/17 satellite data, which is publicly accessible in Amazon S3.

## NAS

This project is meant to be deployed both locally and on NASA's NEX Super Computers.

> 1. For detailed documentation on a number of topics and bugs at NAS please see [here.](https://www.nas.nasa.gov/hecc/support/kb/)

Author: ```suggestion
1. For detailed documentation on a number of topics and bugs at NAS please see [here](https://www.nas.nasa.gov/hecc/support/kb/).
```

> 1. For an overview of many common NAS tasks see [here.](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html)

Author: ```suggestion
1. For an overview of many common NAS tasks see [here](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html).
```

> 1. For an overview of NAS Compute Architecture see [here.](https://www.nas.nasa.gov/hecc/support/kb/hpc-environment-overview_25.html)

Author: ```suggestion
1. For an overview of NAS Compute Architecture see [here](https://www.nas.nasa.gov/hecc/support/kb/hpc-environment-overview_25.html).
```

1. Support can be reached easily by phone at 800-331-8737 and <support@nas.nasa.gov>.

### Storage

1. All data should be stored on Lou. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/the-lou-mass-storage-system_371.html).
1. For all data storage purposes, please store your data on ` Lou:/u/your_nas_username`

> 1. `/nobackup/your_nas_username` is cleared regularly regularly. `home4/your_nas_username` is not.

Author: ```suggestion
1. `/nobackup/your_nas_username` is cleared regularly. `home4/your_nas_username` is not.
```

> 1. NAS has much of the GOES16 data we are working with available in `nex/datapool/goes16`, accessible from pfe.

Author: Maybe expanding what PFE means, or a quick overview of the different compute servers?

> 1. THe `/nasa` folder available via pfe has a lot of useful software you can easily load.

Author: ```suggestion
1. The `/nasa` folder available via pfe has a lot of useful software you can easily load.
```

### Conda on NAS

1. Conda is natively supported on NAS. On a given instance, to ensure that conda is loaded, please call:
`% module use -a /u/analytix/tools/modulefiles` and `% module load miniconda3/v4`.
1. You can create your own conda environments easily, and call them on any interactive or reserved node.

### Interactive Sessions

All users of NAS can launch an interactive PBS session with qsub, with a sample command found here:
`qsub -I -q v100 -l select=1:ncpus=1:model=sky_gpu,walltime=12:00:00`. The node ID (for instance `r101i0n4`) is constant 

> per user (it does not change per different interactive sessions).

Author: when does the `node ID` appear?

> Alternatively, users can launch jobs via a PBS script. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/using-conda-environments-for-machine-learning_557.html.

Author: ```suggestion
Alternatively, users can launch jobs via a PBS script. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/using-conda-environments-for-machine-learning_557.html).
```

A sample script can be found in the examples folder as well.

### Reserved Notes

> All users can reserve a compute nodes. For more information see [here.](https://www.nas.nasa.gov/hecc/support/kb/reserving-a-dedicated-compute-node_556.html)

Author: ```suggestion
All users can reserve a compute node. For more information see [here](https://www.nas.nasa.gov/hecc/support/kb/reserving-a-dedicated-compute-node_556.html).
```

> These reserved nodes are nice ways to setup long-running Jupyter Notebooks. Users may have at most one reserved node.

Author: ```suggestion
These reserved nodes are nice ways to set up long-running Jupyter Notebooks. Users may have at most one reserved node on the Pleaides compute cluster.
```
